pycounter/constants.py

```diff
@@ -237,4 +243,5 @@
     "BR2": u"Total for all titles",
     "BR3": u"Total for all titles",
     "DB2": u"Total for all databases",
+    "MR1": "Total for all collections",
```

Comment:
I wonder why the other strings are marked with 'u'. Is pycounter Python 2 compatible?

Reply:
I think the `master` is not compatible, but `2.x` branch is.